roamance/src/utils/index.ts

```
@@ -0,0 +1,21 @@
+    month: 'long',
+    day: 'numeric',
+    year: 'numeric',
+  });
+}
+
+/**
+ * Truncate a string to a specified length
+ */
+export function truncateString(str: string, length: number): string {
+  if (str.length <= length) return str;
+  return str.slice(0, length) + '...';
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)